app/models/registration.rb

```diff
@@ -0,0 +1,2 @@
+class Registration < ApplicationRecord
```

Comment:
Registration là để làm gì thế?

Reply:
em định làm Thanh toán online bằng PayPal